hack/inject-cmdline-to-template.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Please add comment lines to explain what this script does.
Also please squash commigs.

Reply:
done